CHANGELOG.md

```diff
@@ -1,6 +1,49 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+
+## [2.84.0]
+### Added
+- Contacts Support Added for Legal Entity, Service Agreement and Users 
+- Usage Sample of Bootstrap json to be added to Legal Entity, Service Agreement and User 
+```yaml
```

Comment:
The content is still in `json` but formatting is mentioned as `yaml`. Please change the content from `json` to `yaml`

Reply:
done, check again and approve